stl/inc/valarray

```diff
@@ -983,8 +1504,8 @@ public:
 
     size_t _Off(_Sizarray& _Indexarr) const {
         size_t _Idx, _Ans = _Start;

```

Comment:
Note to self: there will be a trivial adjacent-line merge conflict with #1226 here. No change requested.